candidate-snaps-review/candidate.yml

```diff
@@ -4,7 +4,8 @@ chromium: []
 eog: []
 epiphany: []
 evince: []
-firefox: []
+firefox:
```

Comment:
Please, don't mix patches. For this time I'll pass it, but it's not a good idea.